data/test_evaluations/retrieval_intent_story.yml

```diff
@@ -0,0 +1,12 @@
+stories:
+- story: chitchat name
+  steps:
+  - user: |
+      What is your name?
+    intent: chitchat/ask_name
+  - action: utter_chitchat
+  - user: |
+      What's the weather like today?
+    intent: chitchat
+  - action: utter_chitchat
+  
```

Comment:
```suggestion

```

Reply:
I think this file should be in `data/test_yaml_stories` folder instead.